src/middlewared/middlewared/etc_files/initramfs_update.py

```diff
@@ -0,0 +1,7 @@
+from middlewared.utils import run
+
+
+async def render(service, middleware):
+    await run(
+        '/usr/local/bin/truenas-initrd.py', '/', encoding='utf8', errors='ignore', check=False
```

Comment:
I don't understand why this is an etc plugin and not a `boot` service method like it was before, what configuration are we generating here?